docs/java-rest/high-level/apis/search.asciidoc

```diff
@@ -21,7 +21,7 @@ which contains everything that
 in the Rest API would be placed in the search request body.
```

Comment:
I was trying to make the connection from sb. using the Java API (which is then turned into a REST request under the hood) vs. sending the Rest request explicitly. We mostly document the rest syntax in the docs. But I can see how this is confusing here, will change.